index-fr.html

```diff
@@ -691,7 +690,7 @@ <h2>Crédits et références</h2>
 
     <p>
       Voilà <a href="https://static1.squarespace.com/static/5e8126f89327941b9453eeef/t/5f2c4463a5c9f75a38d2b26f/1596736614213/N95DECON_cloth_mask_breathability_filtrationtechnical_report_v1_200804.pdf">quelques</a> <a href="https://www.maskfaq.com/test-results">ressources</a> <a href="http://built-envi.com/what-kind-of-mask-should-i-be-wearing-to-protect-against-covid-19/">utiles</a> sur les capacités de filtrage des matériaux couramment utilisés dans les masques.
-      Bien que les masques aient une efficacité variable selon le matériau et l'ajustement au visage, certains matériaux peuvent <a href="https://static1.squarespace.com/static/5e8126f89327941b9453eeef/t/5f2c4463a5c9f75a38d2b26f/1596736614213/N95DECON_cloth_mask_breathability_filtrationtechnical_report_v1_200804.pdf">dépasser</a> <a href="https://www.maskfaq.com/test-results">80%</a> d'efficacité de filtrage lorsqu'ils sont utilisés dans un masque fermement ajusté.
+      Bien que les masques aient une efficacité variable selon le matériel et l'ajustement au visage, certains matériaux peuvent <a href="https://static1.squarespace.com/static/5e8126f89327941b9453eeef/t/5f2c4463a5c9f75a38d2b26f/1596736614213/N95DECON_cloth_mask_breathability_filtrationtechnical_report_v1_200804.pdf">dépasser</a> <a href="https://www.maskfaq.com/test-results">80%</a> d'efficacité de filtrage lorsqu'ils sont utilisés dans un masque fermement ajusté.
```

Comment:
Here we are table about the kind of fabric, so "matériau" seems more adequate. This is also consistent with the rest of the sentence: "certains matériaux...".